src/Microsoft.DocAsCode.Build.Engine/MarkdownProviders/DfmJsonTokenTreeServiceProvider.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.Engine
+{
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using System.Composition;
+
+    using Microsoft.DocAsCode.Dfm;
+    using Microsoft.DocAsCode.MarkdownLite;
+    using Microsoft.DocAsCode.Plugins;
+
+    [Export("dfmJsonTokenTree", typeof(IMarkdownServiceProvider))]
+    public class DfmJsonTokenTreeServiceProvider : IMarkdownServiceProvider
```

Comment:
use `GfmBuilder` for GFM.

---
In reply to: [77466039](https://github.com/dotnet/docfx/pull/667#discussion_r77466039) [](ancestors = 77466039,77465842)